modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/SalesRequestForm/SalesRequestForm.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {useState} from 'react';
+
+import PRMFormik from '../../common/components/PRMFormik';
+import DealRegistration from '../../common/interfaces/dealRegistration';
+import {StepType} from './enums/stepType';
+import General from './steps/General';
+import Review from './steps/Review';
+
+type StepComponent = {
+	[key in StepType]?: JSX.Element;
+};
+
+const SalesRequestForm = () => {
```

Comment:
```suggestion
const DealRegistrationForm = () => {
```
Lembre de alterar o nome da pasta e do arquivo.